README.md

```diff
@@ -9,16 +9,20 @@ A live demo is available at [`http://daire.cs.uwaterloo.ca/`](http://daire.cs.uw
 This repo holds the code that runs that demo.
 
 ## Installation
-If you haven't set up Archives Unleashed Toolkit, follow the instructions [here](https://aut.docs.archivesunleashed.org/docs/home).
+If you haven't set up the Archives Unleashed Toolkit, follow the instructions [here](https://aut.docs.archivesunleashed.org/docs/home).
 
 Use the Toolkit to [extract image information](https://aut.docs.archivesunleashed.org/docs/aut-spark-submit-app#image-information) and place the parquet files in `data/images/`:
 ```sh
-spark-submit --class io.archivesunleashed.app.CommandLineAppRunner path/to/aut-fatjar.jar --extractor ImageInformationExtractor --input /path/to/warcs/* --output /path/to/daire/data/images --output-format parquet
+spark-submit --class io.archivesunleashed.app.CommandLineAppRunner path/to/aut-fatjar.jar \ 
+--extractor ImageInformationExtractor --input /path/to/warcs/* \
```

Comment:
I'm being pedantic, but an indent on the second line makes the command look prettier.